src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -496,7 +497,6 @@ class ChatSwitcherView extends React.Component {
         } else if (this.state.search && this.state.options.length === 0) {
             feedbackMessage = 'Don\'t see who you are looking for? Type their valid email/phone number to invite them.';
         }
-
```

Comment:
There should be no changes in this file now all the whitespace edits can go please.